src/middleware/node/middleware.ts

```diff
@@ -47,7 +47,10 @@ export async function middleware(
   // Check if the Content-Type header is `application/json` and allow for charset to be specified in it
   // Otherwise, return a 415 Unsupported Media Type error
   // See https://github.com/octokit/webhooks.js/issues/158
-  if (!request.headers["content-type"].startsWith("application/json")) {
+  if (
+    request.headers["content-type"] == null ||
```

Comment:
why not check if it's falsy?
```suggestion
    !request.headers["content-type"] ||
```

Reply:
Because I am bouncing between too many languages!  You are absolutely correct.